Add SES Client

MancunianSam opened the pull request.

MancunianSam committed:
Add SES Client

TomJKing reviewed (commented):
Looks good. Is it worth adding some unit tests for the new SES client? Also the README will need updating with details of the new SES Client

MancunianSam committed:
Use the client passed to the method.

MancunianSam committed:
Add test for the sendEmail method

MancunianSam committed:
Update README

MancunianSam commented:
Good point on the unit test. I've added one in. I've updated the README as well. I've put something in there about the SQS stuff as well because I didn't update the README whenever I did that.

MancunianSam committed:
Setting version to 0.1.4

MancunianSam committed:
Setting version to 0.1.5-SNAPSHOT

TomJKing commented on README.md:
```diff
@@ -4,5 +4,12 @@ This is a repository for useful methods and circe decoders. These can be used wh
+* SQS Utils - There is a send and a delete method for sqs messages.
+* SES Utils - There is a send email method.
+
+There are also clients for AWS services. These are configured with an http client, the region and and an endpoint configurable in an aplication.conf file. There are:
```
Double "and": "... the region and and an endpoint ..."

MancunianSam replied:
I think I was getting warnings about that before I switched them off thinking I know better :smile:

TomJKing commented on README.md:
```diff
@@ -4,5 +4,12 @@ This is a repository for useful methods and circe decoders. These can be used wh
+There are also clients for AWS services. These are configured with an http client, the region and and an endpoint configurable in an aplication.conf file. There are:
+* An SQS Client
+* An S3 Client
+* As SES Client
```
Should be "An" not "As"

MancunianSam replied:
I've changed this one as well.

TomJKing reviewed (commented):
Just a couple of typos in the README

MancunianSam committed:
Corrections to the README
